docs/index.md

```diff
@@ -158,8 +158,26 @@ When an issuer generates a new key to sign Health Cards, the public key SHALL be
 issuer's JWK set in its `jwks.json` file. Retired private keys that are no longer used to sign Health Cards SHALL be destroyed.
 Older public key entries that are needed to validate previously
 signed Health Cards SHALL remain in the JWK set for as long as the corresponding Health Cards
-are clinically relevant. However, if a private signing key is compromised, then the issuer SHALL immediately remove the corresponding public key
-from the JWK set in its `jwks.json` file and request revocation of all X.509 certificates bound to that public key.
+are clinically relevant. However, if a private signing key is compromised, then the issuer SHALL immediately remove the corresponding public key from the JWK set in its `jwks.json` file and request revocation of all X.509 certificates bound to that public key; verifiers will from then on reject all Health Cards signed using that key.
+
+### Revocation
+
+Individual Health Cards MAY be revoked by an issuer using the digest of their JWS representation. To revoke a card, an issuer:
+1. Calculates the base64url-encoded SHA-256 `<<digest>>` of the utf-8 encoding of the `<<Health Card as JWS>>`
+2. Add the digest to their `https://"<<Issuer URL>>"/.well-known/revoked-shcs-by/kid/"<<kid>>".json`, where
+   - `"<<Issuer URL>>"` is the issuer URL listed in the Health Card,
+   - `"<<kid>>"` is the key ID with which the Health Card was signed,
+   - `"<<kid>>".json" is a JSON file containing an array of revoked Health Cards digests:
+   ```json
+   {
+   "kid": "<<kid>>",
+   "digests": [...,<<digest>>]
```

Comment:
Thinking about whether we could keep the size of these files manageable women in verifications is large. We could suggest things like taking only the first 128 bits (or fewer bits, or any unique prefix) of a sha256 digest without losing our ability to uniquely identify cards. The client logic would still be simple enough, which is to say as long as you have a fresh revocation file, you can just look for any value in this array that is a prefix of the sha256 of the card you have in hand.